conn/i2c/i2c_test.go

```diff
@@ -128,3 +128,21 @@ func TestAddr_Set(t *testing.T) {
 		}
 	}
 }
+
+func TestAddr_String(t *testing.T) {
+	tests := []struct {
+		Addr Addr
+		want string
+	}{
+		{0x01, "0x1"},
+		{0x24, "0x24"},
+		{0x3ff, "0x3ff"},
+	}
+
+	for _, tt := range tests {
+		got := tt.Addr.String()
```

Comment:
This can be inlined in the next line as `if got := tt.Addr.String(); got != tt.want {`